server/src/main/scala/test/TestDrtSystem.scala

```diff
@@ -72,7 +72,7 @@ class TestDrtSystem(override val actorSystem: ActorSystem, override val config:
       val cs = startCrunchSystem(None, None, None, None, true)
       subscribeStaffingActors(cs)
       startScheduledFeedImports(cs)
-      cs.killSwitches
+//      cs.killSwitches
```

Comment:
delete or uncomment.

Reply:
done - the compiler was warning it doesn't do anything so i tried with it commented out and forgot to delete it..